packages/frontend/src/hooks/useAssets.ts

```diff
@@ -4,23 +4,23 @@ import { hopAppNetwork } from 'src/config'
 import logger from 'src/logger'
 import Network from 'src/models/Network'
 
-export function useAssets(selectedBridge?: HopBridge, fromNetwork?: Network, toNetwork?: Network) {
+export function useAssets(selectedBridge?: HopBridge, network?: Network, toNetwork?: Network) {
```

Comment:
minor suggestion;

I think the previous var name `fromNetwork` was more descriptive than having `network` because there is also a `toNetwork`

Reply:
i changed it to `network` here because this hook could technically work with just one network (both the `fromNetwork` and the `toNetwork`), which is the case for [`/convert`](https://github.com/hop-protocol/hop/blob/09b876b84df96874613044bb82e384a1a5e0eb05/packages/frontend/src/pages/Convert/ConvertContext.tsx#L129) and `/pool`. but i also see it from your point of view. lemme know what you think ^^ @miguelmota

Reply:
that sounds good to me,
no strong opinion